src/clr-angular/forms/select/select.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component, ElementRef, OnDestroy, ViewChild} from "@angular/core";
+import {IfOpenService} from "../../utils/conditional/if-open.service";
+import {Subscription} from "rxjs/Subscription";
+import {POPOVER_HOST_ANCHOR} from "../../popover/common/popover-host-anchor.token";
+
+@Component({
+    selector: "clr-select",
+    templateUrl: "select.html",
+    providers: [
+        IfOpenService,
+        {provide: POPOVER_HOST_ANCHOR, useExisting: ElementRef
+    }],
+    host: {"[class.clr-select]": "true"}
+})
+export class ClrSelect implements OnDestroy {
+    open: boolean;
```

Comment:
Should this be `private`? What's this property for? We assign to it in the `this._ifOpenService.openChange`, but didn't use it anywhere else.

Reply:
Now I am thinking this variable and the subscription are actually not doing anything. They were required before but @youdz asked me to move things around. So I think I missed removing them during the cleanup. Will take a look at this. Thanks!